ProjectManagmentSystem/src/Modules/ProjectsModule/components/ProjectsList/ProjectsList.tsx

```diff
@@ -1,7 +1,307 @@
-import React from 'react'
-
+import React, { useContext, useEffect, useState } from "react";
+import Styles from "./ProjectsList.module.css";
+import axios from "axios";
+import { AuthContext } from "../../../Context/AuthContext";
+import NoData from "../../../SharedModule/components/NoData/NoData";
+import { useNavigate } from "react-router-dom";
+import Button from "react-bootstrap/Button";
+import Modal from "react-bootstrap/Modal";
+import DeleteData from "./../../../SharedModule/components/DeleteData/DeleteData";
+import { useToast } from "../../../Context/ToastContext";
+import Loading from "../../../SharedModule/components/Loading/Loading";
 export default function ProjectsList() {
+  const { requestHeaders, baseUrl }: any = useContext(AuthContext);
+  const { getToast } = useToast();
+  const [projectsList, setProjectsList] = useState([]);
+  const [ProjectId, setProjectId] = useState(0);
+  const [isLoading, setIsLoading] = useState(false);
+  const [employeeId, setEmployeeId] = useState("");
+  const [showDelete, setShowDelete] = useState(false);
+  const [titleValue, setTitleValue] = useState("");
+  const [statusValue, setStatusValue] = useState("");
+  const [arrayOfPages, setArrayOfPages] = useState<number[]>([]);
+  const [currentPage, setCurrentPage] = useState(1);
+  const [pageSize, setPageSize] = useState(5);
+  const [showListSize, setShowListSize] = useState(false);
+  const [totalNumberOfRecords, setTotalNumberOfRecords] = useState<number[]>(
+    []
+  );
+  const handleDeleteClose = () => setShowDelete(false);
+  const handleDeleteShow = (id: number) => {
+    setProjectId(id);
+    setShowDelete(true);
+  };
+  const navigate = useNavigate();
+
+  ////API's
+
+  //Get ALL Projects API
+  const getProjectsList = async ( title: string,
+    status: string,
+    pageSize: number,
+    pageNumber: number) => {
+    try {
+      let {data} = await axios.get(`${baseUrl}/project/manager/?pageSize=${pageSize}&pageNumber=${pageNumber}`,
```

Comment:
We can include the pageSize and number in the params object.

Reply:
Done in different branch